tempesta_fw/http.h

```diff
@@ -152,6 +152,8 @@ typedef enum {
 	TFW_HTTP_HDR_HOST,
 	TFW_HTTP_HDR_CONTENT_LENGTH,
 	TFW_HTTP_HDR_CONTENT_TYPE,
+	TFW_HTTP_HDR_USER_AGENT,
+	TFW_HTTP_HDR_COOKIE,
```

Comment:
Please add to the comment above that Cookie is singular header due to RFC 6265 5.4. Also please find and document the requirement for User-Agent.